src/utils/components/Consoles/components/vnc-console/VncConsole.tsx

```diff
@@ -175,16 +176,19 @@ export const VncConsole: React.FC<VncConsoleProps> = ({
       )}
       <div className={css(styles.consoleVnc)}>
         {children}
-        {status === disconnected && (
-          <EmptyState>
-            <EmptyStateBody>
-              {textDisconnected || t('Click Connect to open the VNC console.')}
-            </EmptyStateBody>
-            <Button variant="primary" onClick={connect}>
-              {textConnect || t('Connect')}
-            </Button>
-          </EmptyState>
-        )}
+        {status === disconnected &&
+          (CustomConnectComponent ? (
```

Comment:
do we plan to use this component with a different way to re-connect ?

Reply:
yes, console tab is different. and in the future, it's possible other places if needed will be custom as well.